ui/selectors/confirm-transaction.js

```diff
@@ -140,8 +141,8 @@ export const use4ByteResolutionSelector = (state) =>
   state.metamask.use4ByteResolution;
 export const currentCurrencySelector = (state) =>
   state.metamask.currentCurrency;
-export const conversionRateSelector = (state) => state.metamask.conversionRate;
-
+export const conversionRateSelector = (state) =>
```

Comment:
isn't this a duplicate of the `getConversionRate` selector in `ducks/metamask/metamask.js`. I guess there are a bunch of duplicates in here?